internal/app/machined/pkg/controllers/k8s/kubelet_spec.go

```diff
@@ -271,18 +270,11 @@ func NewKubeletConfiguration(cfgSpec *k8s.KubeletConfigSpec) (*kubeletconfig.Kub
 	config.KubeletCgroups = constants.CgroupKubelet
 	config.RotateCertificates = true
 	config.ProtectKernelDefaults = true
+	config.ContainerRuntimeEndpoint = "unix://" + constants.CRIContainerdAddress
 
+	// SeccompDefault feature gate is enabled by default Kubernetes 1.25+, GA in 1.27
```

Comment:
no need to enable feature gate anymore (enabled in 1.25+), and kubelet complains about trying to enable GA flag